### ISSUE TITLE
DoTA dataset

### ISSUE THREAD

**Issue by wangzhipeng2020** [MoonBlvd/Detection-of-Traffic-Anomaly#19]
Can I run the DoTA dataset in your tad-IROS2019 project?@MoonBlvd

**Comment by MoonBlvd** [comment]
Yes you can! The DoTA paper is actually an extention of the TAD paper!

**Comment by ghadeerAbdulmajeed** [comment]
Hi @wangzhipeng2020 did you mange to run the DoTA dataset on the tad-IROS2019 project?
